OfficeDevPnP.Core/OfficeDevPnP.Core/Framework/Provisioning/ObjectHandlers/ObjectPages.cs

```diff
@@ -69,6 +69,12 @@ public override void ProvisionObjects(Web web, ProvisioningTemplate template)
                     web.AddWikiPageByUrl(url);
                     web.AddLayoutToWikiPage(page.Layout, url);
                 }
+
+                if (page.WelcomePage)
+                {
+                   web.SetHomePageByServerRelativePath(url);
```

Comment:
Hi Erwin,

I have just committed the way you proposed. Thanks again for the review! :)